test/RunTests

```diff
@@ -821,12 +821,12 @@ exit $ErrorCount;
 __DATA__
 # Test 1:
 {VW} -k -l 20 --initial_t 128000 --power_t 1 -d train-sets/0001.dat \
-    -f models/0001.model -c --passes 8 --invariant \
+    -f models/0001_1.model -c --passes 8 --invariant \
```

Comment:
Why the name change?

Reply:
I wanted to keep it consistent with other name changes.  I append "_<test#>" when I notice an issue.